app/policies/user_policy.rb

```diff
@@ -33,4 +33,8 @@ def update?
   def destroy?
     @current_user.superadmin? && !@user.superadmin?
   end
+
+  def reset_password?
+    @current_user.superadmin? || @current_user == @user
+  end
```

Comment:
Just realized this is the same as `update?` above, is there a reason we need this?